<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadAlunos</class>
 <widget class="QMainWindow" name="CadAlunos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Alunos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Matricula:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_matricula">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>27</y>
      <width>161</width>
      <height>19</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_nome">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>64</y>
      <width>156</width>
      <height>19</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>57</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_idade">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>97</y>
      <width>159</width>
      <height>19</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Idade:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>122</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Curso:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_novo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Novo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_editar">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>169</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_excluir">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tb_alunos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>451</width>
      <height>131</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Matricula</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nome</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Idade</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Curso</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>343</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Buscar (Nome):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_buscar">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>349</y>
      <width>347</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_curso">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>126</y>
      <width>159</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Dev Fullstack</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Design</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fotografia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Markeing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Metaverso</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
